include/nautilus/nautilus.h

```diff
@@ -140,10 +143,6 @@ do {									\
 } while (0)
 
 
-void panic(const char *, ...) __attribute__((noreturn));
-    
-#define panic(fmt, args...)         panic("PANIC at %s(%d): " fmt, __FILE__, __LINE__, ##args)
```

Comment:
Not sure why these were here since they are in `printk.h` as well, I didn't really look into why but this duplication started breaking things when I started pulling apic out of various places